simple_database/components.py

```diff
@@ -0,0 +1,127 @@
+import csv
+
+class DbClass(object):
+    """
+    Returns a database object.
+    """
+    def __init__(self, name):
+        self.name = name
+        self.filename = name + '.csv'
```

Comment:
Hardcoded string, always better to have it in a constant/variable DEFAULT_EXTENSION